src/email/mailchimp/custom_segmentation.R

```diff
@@ -31,18 +31,18 @@ box::use(../../utils/get_env[get_env])
 #'
 #' @param indicator_id Unique indicator ID
 #' @param iso3 Vector of ISO3 codes to generate conditions for
-#' @param test Whether or not this is a test email. If so, only sends to contacts
+#' @param dry_run Whether or not this is a dry run email. If so, only sends to contacts
 #'     with the `hdx-signals-test` tag in Mailchimp.
```

Comment:
Should also change this tag in Mailchimp?

Reply:
Yeah good point! How would the mechanics of that work?